docs/sql-ref-syntax-dml-load.md

```diff
@@ -19,4 +19,63 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+`LOAD DATA` loads data from a directoy or a file into a table or into a partition in the table. A partition spec should be specified whenever the target table is partitioned. The `LOAD DATA` statement can only be used with tables created using the Hive format.
```

Comment:
@huaxingao What if we say it this way  ? Please feel free to change it the way you see fit.

LOAD DATA statement loads the data into a table from the user specified directory or file. If a directory is specified then all the files from the directory are loaded. If a file is specified then only the single file is loaded. Additionally the `LOAD` statement takes an optional partition specication. When a `partiion-spec` is specified, the data files (when input source is a directory) or the single file (when input source is a file) are loaded into the target table.